packages/hop-node/src/watchers/CommitTransferWatcher.ts

```diff
@@ -19,12 +19,16 @@ export interface Config {
   dryMode?: boolean
 }
 
+interface LastCommitForChainId {
+  sentCommitTx: boolean
+}
+
 const BONDER_ORDER_DELAY_MS = 60 * 1000
 
 class CommitTransfersWatcher extends BaseWatcherWithEventHandlers {
   siblingWatchers: { [chainId: string]: CommitTransfersWatcher }
-  minPendingTransfers: number = 1
   minThresholdAmount: BigNumber = BigNumber.from(0)
+  lastCommitForChainId: { [chainId: number]: LastCommitForChainId } = {}
```

Comment:
The LastCommitForChainId only contains one property, so we can simplify by doing

```js
sentCommitTxForChainId: { [chainId: number]: boolean } = {}
...
 if (sentCommitTxForChainId[destinationChainId]) {
   ...
   return
 }
 ...
 this.sentCommitTxForChainId[destinationChainId] = true
```